software/controller/lib/core/comms.cpp

```diff
@@ -1,8 +1,7 @@
 #include "comms.h"
 
-#include "algorithm.h"
 #include "hal.h"
-#include "network_protocol.pb.h"
+#include <algorithm>
```

Comment:
Remind me, why is the STL algs library now ok while it was not before?

Reply:
It is not supported by the STM32duino we used at first for the Nucleo board, while it is fully supported by the ST STM32 platform we migrated to when the current HAL became usable.